ts/features/bonus/cgn/components/merchants/CgnMerchantCategoryItem.tsx

```diff
@@ -21,7 +21,8 @@ const styles = StyleSheet.create({
   body: {
     borderRadius: 8,
     marginBottom: 10,
-    width: widthPercentageToDP("42.13%")
+    width: widthPercentageToDP("42.13%"),
+    minHeight: 121
```

Comment:
Since now we have a maxHeigth, could be worth to add `<H2 color={"white"} numberOfLines={props.child === undefined ? 3 : 2}>` in order to have the ellipsis with a very long text, or the actual behaviour is intentional?




Actual            |  Proposed
:-------------------------:|:-------------------------:
<img width="380" alt="Schermata 2022-02-15 alle 17 38 08" src="https://user-images.githubusercontent.com/26501317/154107030-e4728acb-8bbf-41e2-a39c-c32233529d51.png"> |  <img width="388" alt="Schermata 2022-02-15 alle 17 37 03" src="https://user-images.githubusercontent.com/26501317/154106849-c81e48ec-8a9f-4d58-850f-8208f585c5da.png">

Reply:
We actually have a `minHeight` not a maxHeight this only for cases like `home` category which coming on first row would be shorter than the others

Reply:
You're right, I did something wrong during the test 🤣 !